graphql/version_resolver.go

```diff
@@ -297,15 +298,16 @@ func (r *versionResolver) Tasks(ctx context.Context, obj *restModel.APIVersion,
 	}
 
 	opts := task.GetTasksByVersionOptions{
-		Statuses:                       getValidTaskStatusesFilter(options.Statuses),
-		BaseStatuses:                   getValidTaskStatusesFilter(options.BaseStatuses),
-		Variants:                       []string{variantParam},
-		TaskNames:                      []string{taskNameParam},
-		Page:                           pageParam,
-		Limit:                          limitParam,
-		Sorts:                          taskSorts,
-		IncludeBaseTasks:               true,
-		IncludeEmptyActivation:         utility.FromBoolPtr(options.IncludeEmptyActivation),
+		Statuses:         getValidTaskStatusesFilter(options.Statuses),
+		BaseStatuses:     getValidTaskStatusesFilter(options.BaseStatuses),
+		Variants:         []string{variantParam},
+		TaskNames:        []string{taskNameParam},
+		Page:             pageParam,
+		Limit:            limitParam,
+		Sorts:            taskSorts,
+		IncludeBaseTasks: true,
+		// If the version is a patch, we want to exclude inactive tasks by default.
+		IncludeNeverActivatedTasks:     !evergreen.IsPatchRequester(v.Requester) || utility.FromBoolPtr(options.IncludeEmptyActivation) || utility.FromBoolPtr(options.IncludeNeverActivatedTasks),
```

Comment:
Out of curiosity, why do we have both IncludeEmptyActivation AND IncludeNeverActivatedTasks? Couldn't we just combine these?

Reply:
Oh jk i see it's deprecated

Reply:
Yup, we tend to keep around deprecated endpoints for a little bit of time until we ensure we remove all references to it. This just maps the deprecated field to the new field.